descriptors/ModuleDescriptor-template.json

```diff
@@ -33,6 +33,17 @@
         }
       ]
     },
+    {
+      "id": "item-storage-batch-sync",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/item-storage/batch/synchronous",
+          "permissionsRequired": ["inventory-storage.items.batch.post"]
```

Comment:
If we might introduce an asynchronous batch approach, would we want to constrain access to that with different permissions?

If so, I suggest the permissions for these endpoints be named specifically for the synchronous version.

Reply:
I don't think that there is a need for this. If a user is allowed to POST to one of the batch API endpoints why should the POST to the other batch API endpoints been restricted? They all give access to the same table (for example instance).
And we can easily create subpermissions at a later time if there actually is a need for it.

Reply:
Ok, I'll leave the decision to you. 

I raised the question because I tend to err on the the side of caution with permissions, making them more specific if unsure.

This is due to the UI modules composing these into higher level permission sets, which introduces a subtle coupling which effects how easy it is to change the permissions later.